src/window/NavigationDrawer.qml

```diff
@@ -25,7 +25,7 @@ PopupBase {
     overlayLayer: "dialogOverlayLayer"
     overlayColor: Qt.rgba(0, 0, 0, 0.3)
 
-    width: Math.min(parent.width - 1 * Device.gridUnit * Units.dp, 5 * Device.gridUnit * Units.dp)
```

Comment:
this correction need to be in another branch as it has nothing to do with the toolbar size @bmonio could you please move it ?